mne/bem.py

```diff
@@ -1115,6 +1119,10 @@ def read_bem_solution(fname, verbose=None):
     -------
     bem : dict
         The BEM solution.
+
+    See Also
+    --------
+    write_bem_solution
```

Comment:
Read our revamped manual :)

But really, the `model` downsamples and puts together one or three surfaces with conductances in a FIF file after doing some sanity checks. The `solution` actually computes the information based on those surfaces and conductances that we need for forward computation.